src/shogun/multiclass/CoverTreeKNNSolver.h

```diff
@@ -17,9 +17,9 @@ namespace shogun
 {
 
 /**
- * The CoverTreeKNNSolver class is a solver class which inherit from KNNSolver class. 
- * It use the cover tree to speed up the nearest neighbor search and you can find more
- * detail information on https://en.wikipedia.org/wiki/Cover_tree
+ * The CoverTreeKNNSolver class is a solver class. 
+ * It use the cover tree to speed up the nearest neighbor searching process.
```

Comment:
Mistakes:
 * It uses (third person verbs have an s!)
 * It uses cover trees, or it uses the cover tree algorithm
 * .... to speed up the nearest neighbour search process (not searching process) ... Also it is not a search process, so rather write: to speed up the nearest neighbour computation

It uses cover